x-pack/filebeat/input/cel/input.go

```diff
@@ -101,6 +102,15 @@ func (input) Run(env v2.Context, src inputcursor.Source, crsr inputcursor.Cursor
 	return input{}.run(env, src.(*source), cursor, pub)
 }
 
+// The sanitizeFileName sanitizes characters like ":" and "/" to replace them with "_"
```

Comment:
```suggestion
// sanitizeFileName sanitizes replaces ":" and "/" with "_", removing repeated instances.
```
Also in httpjson.

Reply:
Please fix this.

Reply:
Fixed in https://github.com/elastic/beats/pull/35154/commits/bc3ee57300a9a46a335b4c1ce15f06c3dddc41be

Reply:
The text at the tip of this branch is
```
// The sanitizeFileName replaces ":" and "/" with "_", removing repeated instances.
// The request.tracer.filename may have ":" when a httpjson input has cursor config and
// the macOS Finder will treat this as path-separator and causes to show up strange filepaths.
```

The first line of this should be "sanitizeFileName returns name with ":" and "/" replaced with "_", removing repeated instances." The article is incorrect here and the use of "replaces" (my error too in the original suggestion) is incorrect because it indicates a replacement in `name` which does not happen because `name` is not passed by reference.

Reply:
I get it now :-).. Will fix it..